.all-contributorsrc

```diff
@@ -1989,14 +1989,33 @@
       "contributions": [
         "code"
       ]
-    },{
+    },
+    {
       "login": "stuposluns",
       "name": "stuposluns",
       "avatar_url": "https://avatars.githubusercontent.com/u/23000883?v=4",
       "profile": "https://github.com/stuposluns",
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "charlrvd",
+      "name": "Charles",
+      "avatar_url": "https://avatars.githubusercontent.com/u/17158847?v=4",
+      "profile": "https://github.com/charlrvd",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "aviflombaum",
+      "name": "Avi Flombaum",
+      "avatar_url": "https://avatars.githubusercontent.com/u/4515?v=4",
+      "profile": "http://www.aviflombaum.com",
+      "contributions": [
+        "code"
+      ]
     }
```

Comment:
There's a bunch of unrelated changes like this in the diff, looks like this needs to have a different base

Reply:
Yeah its dependent on the Search inputs work as I thought that would have been merged by now 😅

Reply:
Updated to fix the weird diff.